ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ip-client-source-interfaces.j2

```diff
@@ -1,56 +1,62 @@
 {# eos - ip client source-interfaces #}
-{% if ip_ftp_client_source_interfaces is arista.avd.defined %}
+{% if ip_ftp_client_source_interfaces is arista.avd.defined
```

Comment:
Because I found a bug in the recently implemented code in eos_cli_config_gen that was only visible once you start exercising only parts of the data model. So to ensure that it renders correctly I fixed it here.